expts/hydra-configs/architecture/toymix.yaml

```diff
@@ -79,10 +79,10 @@ datamodule:
     featurization_progress: True
     featurization_backend: "loky"
     processed_graph_data_path:  ${constants.datacache_path}
-    dataloading_from: ram
     num_workers: 30 # -1 to use all
     persistent_workers: False
     featurization:
+      use_graphium_cpp: True
```

Comment:
Sure.  I kept the Python featurization in there mostly for easier comparison between the two for debugging purposes, but if you're happy with the C++ featurization, I can delete the Python featurization.  Deleting it would simplify some things in the calling code, too.

Reply:
I've now deleted the Python featurization, except some tangential bits.  Is it okay to delete graphium/features/properties.py?  It seems like it wasn't being used, and the 2 functions in it were already deprecated.

Reply:
Yes, the file is not used